FlowCrypt/Models/Realm Models/ClientConfigurationRealmObject.swift

```diff
@@ -17,7 +17,7 @@ final class ClientConfigurationRealmObject: Object {
     @Persisted var disallowAttesterSearchForDomains: Data?
     @Persisted var enforceKeygenAlgo: String?
     @Persisted var enforceKeygenExpireMonths: Int = -1
-    @Persisted var user: UserRealmObject!
+    @Persisted var user: UserRealmObject?
```

Comment:
@sosnovsky 
I decided to avoid having force unwrap in Realm objects. Maybe I'm wrong here so please take a look

Reply:
I know force unwrapping is not ideal. It's just that, client configuration always belongs to some user. So I thought it was weird that the user would be optional.

Is there a way to indicate that the user object is required without force unwrapping?

Reply:
Especially, I don't want to have some bugs in the future that would cause realm objects to be saved without reference to the user they belong to - that would be a mess.

Reply:
I think we can have just `@Persisted var user: UserRealmObject` without `?` or `!`, as it means that `user` property is not optional and should be set during `ClientConfigurationRealmObject` initialization.

Reply:
Oh, I missed that Realm needs object properties to be optional. But currently all `ClientConfigurationRealmObject` initializers have required `user` property, so there won't be possibility to create new object without `user` property.

We can also update `var activeUser: UserRealmObject?` to look like
```swift
var activeUser: UserRealmObject {
        guard let user = user else { fatalError() }
        return user
}
```
Then it won't be possible to use configuration without assigned `user`

Reply:
Filed issue #1035